exercises/concept/need-for-speed/src/test/java/NeedForSpeedTest.java

```diff
@@ -195,12 +195,12 @@ public void car_can_finish_with_car_that_just_cannot_finish() {
         int distance = 16;
         var race = new RaceTrack(distance);
 
-        assertThat(race.tryFinishTrack(car)).isFalse();
+        assertThat(race.canFinishRace(car)).isFalse();
     }
 
     @Test
     @Tag("task:6")
-    @DisplayName("The tryFinishCar method returns false when car cannot finish a race")
+    @DisplayName("The  method returns false when car cannot finish a race")
```

Comment:
```suggestion
    @DisplayName("The canFinishRace method returns false when car cannot finish a race")
```

Adding the name back makes it consistent with the other tests.

Reply:
Good spot. Not even sure how that got removed, as I replaced everything...